frontend/Makefile

```diff
@@ -3,6 +3,8 @@ all:
 dev: frontend
 frontend:
 	@echo "Making frontend dev server..."
+	@cd galeri/ && npm run start
 build:
 	@echo "Building frontend..."
+	cd galeri/ && npm run build
```

Comment:
Just get rid of this Makefile 😂 Also, we have some weird nested directories. I think everything in `frontend/galeri/` should be in `frontend/`.